src/ui_fsmenu/load_map_or_game.cc

```diff
@@ -37,11 +37,11 @@ FullscreenMenuLoadMapOrGame::FullscreenMenuLoadMapOrGame()
 void FullscreenMenuLoadMapOrGame::layout() {
 	// UI coordinates and spacers
 	tablex_ = get_w() * 47 / 2500;
-	tabley_ = get_h() * 17 / 50;
+	tabley_ = get_h() * 14 / 50;
 	tablew_ = get_w() * 711 / 1250;
 	tableh_ = get_h() * 6083 / 10000;
 	right_column_x_ = tablex_ + tablew_ + right_column_margin_;
-	buty_ = get_h() * 9 / 10;
+	buty_ = get_h() * 84 / 100;
 	butw_ = (get_w() - right_column_x_ - right_column_margin_) / 2 - padding_;
 	buth_ = get_h() * 9 / 200;
```

Comment:
**tabley_factor**: table relative position (top)
**tableh_factor**: table relative height
**buty_factor**: buttons relative position (top)
**buth_factor**: buttons relative height

For **tabley_factor**, _14/50_ is probably the highest possible value that would allow 3 items to be displayed in a combo box at low resolution.
**tabley_factor** + **tableh_factor** - **buth_factor** = _17/50_ + _6063/10000_ - _9/200_ = _0.9013_ ~= _9/10_ = **buty_factor**
If we change **tabley_factor**, we also need to modify **buty_factor**.
_14/50_ + _6063/10000_ - _9/200_ = _0.8413_ ~= _84/100_